<launch>
    <arg name="rviz" default="false" />

    <node name="camera_pose_publish" pkg="oil_pose_detect" type="camera_pose_publisher.py" respawn="false" output="screen">
        <param name="base_frame" value="world" />
        <param name="camera_frame" value="panda_grasp_frame" />
    </node>

    <!-- start the oil_filler_pose -->
    <node name="detect_oil_pose" pkg="oil_pose_detect" type="detect_oil_with_reconstruct" respawn="false" output="screen">
        <param name="show" value="false" />
        <param name="useExact" value="true" />
        <param name="useCompressed" value="false" />

        <param name="camera" value="realsense" />
        <param name="oil_frame_reference" value="camera_color_optical_frame" />
        <param name="topicColor" value="/camera/color/image_raw" />
        <param name="topicDepth" value="/camera/aligned_depth_to_color/image_raw" />
    </node>

</launch>